build/Dockerfile

```diff
@@ -1,22 +1,34 @@
 FROM dstacademy/steamcmd:0.1
 MAINTAINER DST Academy <support@dst.academy>
 
+# Set build arguments.
+ARG DST_HOME
+ENV DST_HOME ${DST_HOME:-"/opt/dst"}
+ARG DSTA_HOME
+ENV DSTA_HOME ${DSTA_HOME:-"/usr/local/share/dsta"}
+ARG CLUSTER_PATH
+ENV CLUSTER_PATH ${CLUSTER_PATH:-"/var/lib/dsta/cluster"}
+
+# Set environment variables.
+ENV DESCRIPTION="Powered by DST Academy." \
+	SERVER_PORT="10999" \
+	MAX_PLAYERS=4 \
+	GAME_MODE=survival \
+	VOTE_KICK_ENABLE=true \
+	PAUSE_WHEN_EMPTY=true \
+	SHARD_NAME=shard \
+	SHARD_IS_MASTER=false \
+	SHARD_BIND_IP="0.0.0.0" \
+	BACKUP_LOG_COUNT=0
+
```

Comment:
I don't like this. I think that even with 1.0 we must try to reuse as much layers as we can. This way all the pulls would be faster.

I understand that this are more less like variables but...